pandas/conftest.py

```diff
@@ -481,7 +482,7 @@ def tz_aware_fixture(request):
 
 UNSIGNED_INT_DTYPES = ["uint8", "uint16", "uint32", "uint64"]
 UNSIGNED_EA_INT_DTYPES = ["UInt8", "UInt16", "UInt32", "UInt64"]
-SIGNED_INT_DTYPES = [int, "int8", "int16", "int32", "int64"]
+SIGNED_INT_DTYPES = [int, "int8", "int16", "int32", "int64"]  # type: Dtype
```

Comment:
This is actually a `List[Dtype]`

Reply:
I think `Dtype` needs to be updated in pandas._typing to include `Type[Union[float, int]]` for when the built-in values get provided - could you make that change as well?

Reply:
Yep, sure - thanks for the guidance.

Reply:
`Type[Union[float, str, bool]]` would be sufficient as int is duck type compatible with float see https://mypy.readthedocs.io/en/latest/duck_type_compatibility.html#duck-type-compatibility

Reply:
Thanks, both. Just pushed what I _think_ you both meant - let me know. I'm still learning about types! Looks like `complex` was also duck typed to `float`.